lib/charms/nginx_ingress_integrator/v0/nginx_route.py

```diff
@@ -0,0 +1,383 @@
+# Copyright 2023 Canonical Ltd.
+# Licensed under the Apache2.0, see LICENCE file in charm source for details.
+"""Library for the nginx-route relation.
+
+This library contains the require and provide functions for handling
```

Comment:
Do you think would be relevant to add some comment here mentioning that this lib/relation should be used instead of the deprecated ingress one?